src/libs/telemetry/activeSpans.ts

```diff
@@ -0,0 +1,29 @@
+import type {StartSpanOptions} from '@sentry/core';
+import * as Sentry from '@sentry/react-native';
+
+const activeSpans = new Map<string, ReturnType<typeof Sentry.startInactiveSpan>>();
```

Comment:
Isn't this creating a pretty bad precedence? This portion of code (and storing in memory) is very imperative and forces us to start keeping track (also solving bugs) with manually handling this Map. This should all live in the SDK I think and not be handled by the app.